karma.conf.js

```diff
@@ -12,20 +10,28 @@ module.exports = function (config) {
     // Only test IE latest.
     config.browsers.push('internet_explorer_11');
   }
+  var polyOptions = process.argv.filter((val) => {
```

Comment:
Definitely if we're going to add more logic to this. If not, I think it would be a little overkill. Reminds me a bit of this http://m.imgur.com/Q3mkcnl?r